examples/basic_tutorial_tabular.ipynb

```diff
@@ -156,7 +156,7 @@
     "# initialise the pipelien class\n",
     "pipe = Trainer(\n",
     "    model,\n",
-    "    (dl_tr, dl_val),\n",
```

Comment:
Why did you remove the valdiation dataloader?

Reply:
The validation dataloader cannot be transformed in a readable dataloader by the Trainer.
Another solution may be valid as well, if you have any working solution, we'd be happy to use it.

Reply:
@raphaelreinauer has a good point here: if I understood you right @yorickbrunet , your PR is basically removing the feature to chose your validation dataset? Since it was working well before, can you point the cause of this issue?

I believe that before merging we need to fix the issue. It's probably wiser to put back the dl_val in all the examples and have the tests fail, so that we are aware.

Reply:
Actually, it is correct to remove the `dl_val` in other examples as it is `None`.

For this case, `Trainer.train()` expects to read indices from samplers, but 'SequentialSampler' object has no attribute 'indices'. So `dl_val` is not a valid dataloader.
It may have worked before but we change the test https://github.com/reds-heig/giotto-deep/blob/parallel_training/gdeep/trainer/trainer.py#L925 (was == 3, is now >= 2) to have it more coherent (why ask for dataloaders if they are not used?). Thus now the other branch of the if statement is taken, which revealed the issue (which existed before but the dataloader wasn't checked).
Now, to cope with changes at https://github.com/reds-heig/giotto-deep/blob/parallel_training/gdeep/trainer/trainer.py#L925, the dataloader's sampler must have indices.

Reply:
thank you!